homeassistant/components/overkiz/climate_entities/atlantic_electrical_towel_dryer.py

```diff
@@ -0,0 +1,118 @@
+"""Support for Atlantic Electrical Towel Dryer."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    PRESET_BOOST,
+    PRESET_NONE,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.components.overkiz.coordinator import OverkizDataUpdateCoordinator
+from homeassistant.components.overkiz.entity import OverkizEntity
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+PRESET_DRYING = "drying"
+
+OVERKIZ_TO_HVAC_MODE: dict[str, str] = {
+    OverkizCommandParam.EXTERNAL: HVACMode.HEAT,  # manu
+    OverkizCommandParam.INTERNAL: HVACMode.AUTO,  # prog
+    OverkizCommandParam.STANDBY: HVACMode.OFF,
+}
+HVAC_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_HVAC_MODE.items()}
+
+OVERKIZ_TO_PRESET_MODE: dict[str, str] = {
+    OverkizCommandParam.PERMANENT_HEATING: PRESET_NONE,
+    OverkizCommandParam.BOOST: PRESET_BOOST,
+    OverkizCommandParam.DRYING: PRESET_DRYING,
+}
+
+PRESET_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_PRESET_MODE.items()}
+
+
+class AtlanticElectricalTowelDryer(OverkizEntity, ClimateEntity):
+    """Representation of Atlantic Electrical Towel Dryer."""
+
+    _attr_hvac_modes = [*HVAC_MODE_TO_OVERKIZ]
+    _attr_preset_modes = [*PRESET_MODE_TO_OVERKIZ]
+    _attr_temperature_unit = TEMP_CELSIUS
+
+    def __init__(
+        self, device_url: str, coordinator: OverkizDataUpdateCoordinator
+    ) -> None:
+        """Init method."""
+        super().__init__(device_url, coordinator)
+        self.temperature_device = self.executor.linked_device(7)
+
+        self._attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
```

Comment:
Can be move out of the builder.

Reply:
What do you mean? It is not possible to move this out of the `__init__`, since two lines further we need to modify it again.

```Instance of 'AtlanticElectricalTowelDryer' has no '_attr_supported_features' memberpylint(no-member)```. Otherwise I will get.